bpf/lib/common.h

```diff
@@ -1069,9 +1068,8 @@ struct ct_state {
 	      syn:1,
 	      proxy_redirect:1,	/* Connection is redirected to a proxy */
 	      from_l7lb:1,	/* Connection is originated from an L7 LB proxy */
-	      auth_required:1,
```

Comment:
shouldn't this be changed to ` reserved:1,`? Otherwise the `from_tunnel` will be in the wrong place

Reply:
I'll defer to @YutaroHayakawa then.

Reply:
Changed to `reserved1:1,` to avoid moving `from_tunnel` around.